mesonbuild/msubprojects.py

```diff
@@ -216,7 +216,7 @@ def git_stash(self) -> None:
             # Note: `--all` is used, and not `--include-untracked`, to prevent
             # a potential error if `.meson-subproject-wrap-hash.txt` matches a
             # gitignore pattern.
-            self.git_verbose(['stash', 'push', '--all', ':!/.meson-subproject-wrap-hash.txt'])
+            self.git_verbose(['stash', 'push', '--all', ':!/.meson-subproject-wrap-hash.txt', '.'])
```

Comment:
Could be worth adding a comment above telling why the extra dot, or just link to this PR.

Reply:
Other than that, LGTM, thanks.